src/frontend/src/styles/main.css

```diff
@@ -387,10 +385,21 @@ body {
 a,
 .t-link {
   display: inline;
-  color: var(--rc-interaction-text);
   color: inherit;
-  text-decoration: underline;
   cursor: pointer;
+  background-image: linear-gradient(
+    to right,
+    var(--vc-brand-pink),
+    var(--vc-brand-pink) 50%,
+    #000 50%
+  );
+  background-size: 200% 100%;
+  background-position: -100%;
+  position: relative;
+  -webkit-background-clip: text;
```

Comment:
Since background-clip only works in webkit. What happens in other browser?

Reply:
background-clip text is supported cross browser ([source](https://caniuse.com/?search=background-clip)).